old version of gradle has missing dependencies

Just today, our build started failing with the following error:
```
FAILURE: Build failed with an exception.
  
* What went wrong:
A problem occurred configuring project ':react-native-svg'.
> Could not resolve all artifacts for configuration ':react-native-svg:classpath'.
   > Could not find com.android.tools:common:25.2.3.
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/common/25.2.3/common-25.2.3.pom
         https://jcenter.bintray.com/com/android/tools/common/25.2.3/common-25.2.3.jar
         https://maven.google.com/com/android/tools/common/25.2.3/common-25.2.3.pom
         https://maven.google.com/com/android/tools/common/25.2.3/common-25.2.3.jar
     Required by:
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools.build:manifest-merger:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools.ddms:ddmlib:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools.analytics-library:shared:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools.analytics-library:tracker:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools:sdklib:25.2.3> com.android.tools.layoutlib:layoutlib-api:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools:sdklib:25.2.3> com.android.tools:dvlib:25.2.3
         project :react-native-svg > com.android.tools.build:gradle:2.2.3 > com.android.tools.build:gradle-core:2.2.3 > com.android.tools.build:builder:2.2.3 > com.android.tools:sdklib:25.2.3> com.android.tools:repository:25.2.3

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
```
Upgrading to gradle 3.2.1 fixes the issue.
Will submit a PR soon.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -14,7 +14,7 @@ buildscript {
 
     dependencies {
         //noinspection GradleDependency
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.2.1'
     }
 }
 

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -5,6 +5,7 @@ def safeExtGet(prop, fallback) {
 buildscript {
     repositories {
         jcenter()
+        google()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
@@ -37,6 +38,7 @@ android {
 repositories {
     mavenLocal()
     jcenter()
+    google()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
         url "$projectDir/../../../node_modules/react-native/android"

